tests.py

```diff
@@ -520,6 +520,30 @@ def test_multi_ifo_fails(self):
                 **self.signal_parameters,
             )
 
+    def test_makefakedata_usecached(self):
```

Comment:
I think the only difference is that it uses `self.writer` instead of `self.Writer`, so I can use LineWriter there.

Reply:
isn't that what we have `writer_class_to_test` for?

Reply:
Yes... although apparently not for `test_makefakedata_usecached`: https://github.com/PyFstat/PyFstat/blob/82fa81f30aa4db21d15558b75500245eedbf00ec/tests.py#L233

Reply:
Because for some reason (probably historical) this method is using the `Writer` created during start up, rather than generating a new one for this specific test.